easybuild/easyconfigs/n/NCL/NCL-6.4.0-intel-2018a.eb

```diff
@@ -0,0 +1,47 @@
+name = 'NCL'
+version = '6.4.0'
+
+homepage = 'http://www.ncl.ucar.edu'
+description = "NCL is an interpreted language designed specifically for scientific data analysis and visualization."
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'cstd': 'c99', 'openmp': True, 'pic': True}
+
+# download via https://www.earthsystemgrid.org/dataset/ncl.html
+sources = ['%(namelower)s_ncarg-%(version)s.tar.gz']
+patches = ['NCL-%(version)s_fix-types.patch']
+checksums = [
+    'ebc9ec9538044cc4887c0c46adac8522242528d40c974e0b88b5b5d756f22a17',  # ncl_ncarg-6.4.0.tar.gz
+    'f6dfaf95e5de9045745e122cb44f9c035f81fab92f5892991ddfe93945891c8f',  # NCL-6.4.0_fix-types.patch
+]
+
+hdf5_ver = '1.10.1'
+
+dependencies = [
+    ('cURL', '7.55.1'),
+    ('JasPer', '1.900.1'),
+    ('g2lib', '1.4.0'),
+    ('g2clib', '1.6.0'),
+    ('HDF', '4.2.13'),
+    ('HDF5', hdf5_ver), 
+    ('netCDF', '4.6.0', '-HDF5-%s' % hdf5_ver),
+    ('netCDF-Fortran', '4.4.4', '-HDF5-%s' % hdf5_ver),
```

Comment:
no need to customize these for `HDF5` 1.10.1, `netCDF` 4.6.0 with `intel/2018a` already uses `HDF5` 1.10.1

```python
('netCDF', '4.6.0'),
('netCDF-Fortran', '4.4.4'),